tests/jvm/src/test/resources/metac-metacp.expect.diff

```diff
@@ -40,6 +40,16 @@ _root_.com.javacp.
 +language: JAVA
 
 
+============================
+_root_.com.javacp.Interface#
+============================
+--- metac
++++ metacp
+         symbol: "_root_.java.lang.Object#"
+-    declarations: "_root_.com.javacp.Interface#`<init>`()."
```

Comment:
Seems we emit a constructor for java interfaces 🤔

Reply:
https://github.com/scalameta/scalameta/issues/1419

Reply:
Missed that one too!